speechbrain/utils/bleu.py

```diff
@@ -0,0 +1,151 @@
+import sacrebleu
+from speechbrain.dataio.dataio import merge_char, split_word
+from speechbrain.utils.metric_stats import MetricStats
+
+
+def merge_words(sequences):
+    results = []
+    for seq in sequences:
+        words = " ".join(seq)
+        results.append(words)
+    return results
+
+
+def detokenize_batch(detokenizer, sentences):
+    """
+    detokenizer: object
```

Comment:
Short description of what the function does (even if it's clear from the name, it's better to explain it quickly for the users")